next.config.js

```diff
@@ -10,6 +10,14 @@ const nextConfig = {
   experimental: {
     missingSuspenseWithCSRBailout: false,
   },
+  async rewrites() {
+    return [
+      {
+        source: "/api/graphql",
+        destination: "http://localhost:8080/query",
```

Comment:
What happens if the GUAC GQL server is running somewhere else? Should we be checking for the value of `GUACGQL_SERVER_URL` here?

Reply:
See https://github.com/guacsec/guac-visualizer/blob/main/.env#L5  and https://github.com/guacsec/guac-visualizer/commit/b138b9e278120c979228c02b77025ebb7c206ee9

Reply:
Oh wait. I haven't had coffee yet. This is probably what you meant by "The remaining issue in this PR is that the query URL should come from src/config.ts, but this would require converting the next config to typescript." 😄 

So assuming we merge this now, what's the failure case when the server is running somewhere else? It seems like `src/config.ts` already checks for the `GUACGQL_SERVER_URL` environment variable, so we should be able to lean on that, I think. I'm not very familiar with Javascript and friends, I'm more of a sysadmin type, so forgive me if I'm missing something obvious.